MetaMask/metamask-extension
# Part of 17670: Replace Typography with Text swap-import-token

Explanation
Updating Typography to Text import-token under the swap page

Part of: https://github.com/MetaMask/metamask-extension/issues/17670

Screenshots/Screencaps
Before:
![image](https://user-images.githubusercontent.com/33137497/230319929-b2f7dacf-d554-4e37-8df5-44c5b6367403.png)

After:
![image](https://user-images.githubusercontent.com/33137497/230320708-13c02056-cc26-4aae-b239-87580f85da4c.png)

Issue with font-size attribute for component `<Text  variant={TextVariant.bodySm} as="h6"/>`  at .875rem instead of .75rem :
![image](https://user-images.githubusercontent.com/33137497/230318971-c2490c87-a197-41a0-b4a5-822fa05fd543.png)


Manual Testing Steps
Go to the latest build of storybook on this PR
Search ImportToken under Pages/Swaps in the side panel : http://localhost:6006/?path=/story/pages-swaps-importtoken--default-story
Check ImportToken story against develop storybook : https://metamask.github.io/metamask-storybook/?path=/story/pages-swaps-importtoken--default-story

Pre-merge author checklist

Pre-merge reviewer checklist

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the Extension QA Board label.

In this case, a QA Engineer approval will be be required.
